docs/content/porting-unreal-project-to-gdk.md

```diff
@@ -117,17 +117,25 @@ For more information on helper scripts, see [Helper scripts](https://github.com/
         Super::InitPlayerState();
     }
     ```
+    
+1. In the file(s) where you added the `InitPlayerState` function definition, add the following `#include`s:
+
+    ``` cpp
+    #include "SpatialNetDriver.h"
+    #include "EntityId.h"
+    #include "EntityRegistry.h"
+    ```
 
 ## Adding Unreal GDK configurations
 1. In `<GameRoot>\Config`, open `DefaultEngine.ini` and add:
 
     ``` ini
-    [\Script\Engine.Engine]
+    [/Script/Engine.Engine]
```

Comment:
Ah - did I go overboard with updating the forward slashes to back slashes? I thought I'd checked with Michael and only the command `-NetDriverOverrides=/Script/OnlineSubsystemUtils.IpNetDriver` needed forward slashes (but maybe I misunderstood!). So there might be some more rogue backslashes :(

Reply:
I went over the backslashes in this document specifically and they seem to be all good now.
We may need to check the other docs as well.
The forward slashes need to be used everywhere with `/Script/` (*not* `Scripts`, which is a folder)

Reply:
Got it, thanks! I'll add to our docs release checklist - check for rogue back slashes. Hopefully should be easy enough to do a quick search.